modules/pubsub/variables.tf

```diff
@@ -32,6 +32,13 @@ variable "subscription_labels" {
   description = "A set of key/value label pairs to assign to the pubsub subscription or pubsub push subscription."
 }
 
+variable "push_subscription_expiration_ttl" {
+  description = "Pubsub push subscription expiration policy ttl."
+  type        = string
+  default     = null
+  nullable    = true
```

Comment:
@Codasquieves default is null and if user dont pass anything it will be set as null. You dont need to set nullable for this field. Can you plz remove it?